### PR TITLE
update-links-one-full-file

### PR DESCRIPTION
**Pull request by colin-hillis**
Trial for changing links to Hugo links in bulk with regex - one file in one go.

6 links were updated to new format.

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/4.10/community/get-started/get-started-landing.md [hunk 1]
```diff
@@ -40,15 +40,15 @@ If you are already using an open source version of Corda (Corda 4.1–Corda 4.9)
...
+Follow the upgrade guides to make sure your [node]({{< relref "../../community/node-upgrade-notes" >}}) and [CorDapps]({{< relref "../../community/upgrading-cordapps" >}}) are upgraded correctly.
 
 ## Next steps
 
-1. Familiarize yourself with the [Corda key concepts](../about-corda/corda-key-concepts.md). 
-2. Run a [sample CorDapp](../../community/tutorial-cordapp.md) to see Corda in action.
-4. [Build your own CorDapp](../../community/building-a-cordapp-index.md) from scratch. /
+1. Familiarize yourself with the [Corda key concepts]({{< relref "../about-corda/corda-key-concepts" >}}). 
+2. Run a [sample CorDapp]({{< relref "../../community/tutorial-cordapp" >}}) to see Corda in action.
+4. [Build your own CorDapp]({{< relref "../../community/building-a-cordapp-index" >}}) from scratch. /
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/community/get-started/get-started-landing.md:49]:
```suggestion
4. [Build your own CorDapp]({{< relref "../../community/building-a-cordapp-index" >}}) from scratch. 
```

**Review by nadinequinr3 — COMMENTED** [review]
Link on line 33 still needs to be updated.

File: content/en/platform/corda/4.10/community/get-started/get-started-landing.md (hunk 1 above)
Comment (colin-hillis) [reply content/en/platform/corda/4.10/community/get-started/get-started-landing.md]: Thanks for flagging Nadine. No this particular one is still there on purpose. It is one of the other lesser-used link types identified, but this regex doesn't find it. I will write a new one for that type.

**Comment by colin-hillis** [comment]
Thanks for flagging Nadine. No this particular link type is still there on purpose. It is one of the other lesser-used link types identified in our docs, but this regex doesn't find it. I will write a couple of different regex to find the lesser-used ones later, but I have left this one in place now so that I know where one exists to test any new regex with.

**Review by nadinequinr3 — DISMISSED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**